server/images.go

```diff
@@ -543,16 +567,15 @@ func PullModel(name, username, password string, fn func(status, digest string, T
 	total += manifest.Config.Size
 
 	for _, layer := range layers {
-		fn("starting download", layer.Digest, total, completed, float64(completed)/float64(total))
```

Comment:
This and the `download complete` updates are removed in favour of the updates in `downloadBlob`